### PR TITLE
Allow supressing 'bad-procedural-force' errors for compatibility with commercial tools

### PR DESCRIPTION
**Pull request by udif**
See #639 .

### CODE REVIEW

File: scripts/diagnostics.txt [hunk 1]
```diff
@@ -693,6 +692,7 @@ warning event-const EventExpressionConstant "edge expression is constant"
 warning empty-stmt EmptyStatement "extra ';' has no effect"
 warning pointless-void-cast PointlessVoidCast "cast to void for void-returning function '{}' has no effect"
 warning unused-result UnusedResult "ignoring return value of '{}', cast to void to suppress"
+warning bad-procedural-force BadProceduralForce "lvalue of force/release must be a net, a variable, a constant select of a net, or a concatenation of these"
```
Comment (MikePopoloski) [inline scripts/diagnostics.txt:695]: Can you make sure this is included in the default set at the bottom of this file?

**Comment by MikePopoloski** [comment]
Any updates here?

**Comment by udif** [comment]
> Any updates here?

Sorry, due to external circumstances I will be back to this only around the end of the month.

**Comment by MikePopoloski** [comment]
Ok, no problem. Thanks for the update.

**Comment by MikePopoloski** [comment]
Thanks!

**Pull request merged**